### PR TITLE
fix(sdcm/send_email.py): Send email even if no email data found

### PR DESCRIPTION
**Pull request by dkropachev**
https://trello.com/c/53ZOYGgi/1905-send-email-with-result-should-send-email-even-if-file-with-email-results-data-not-found

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

File: sdcm/send_email.py [hunk 1]
```diff
@@ -385,9 +385,37 @@ class PrivateRepoEmailReporter(BaseEmailReporter):
     email_template_file = "results_private_repo.html"
 
 
+class TestAbortedEmailReporter(BaseEmailReporter):
+    _fields = ("repo_ostype",
+               "repo_uuid",
+               "scylla_repo",)
+    email_template_file = "results_private_repo.html"
```
Comment (aleksbykov) [inline sdcm/send_email.py:392]: May be better to rename to test_aborted_email.html?

Comment (dkropachev) [reply sdcm/send_email.py]: Removed it, it is not used there.

**Comment by fruch** [comment]
@dkropachev can we also consider here collecting cases when SCT run didn't even started ?

**Comment by dkropachev** [comment]
@fruch it is exactly how it is implemented, it does not even need test folder.

**Pull request merged**